docs/modules/panel_configuration/pages/index.adoc

```diff
@@ -19,22 +19,116 @@ This section describes the following types of panel configuration:
 [[custom-panel]]
 == Default custom panels
 
-{page-component-title} comes with several pre-configured, custom panels.
-They are described in this section.
+{page-component-title} comes with several pre-configured custom panels:
+
+* *Alarm Histogram:* A bar chart that shows the distribution of alarms by severity level.
++
+image::alarm-histogram.png["An Alarm Histogram panel displaying a horizontal bar graph. The bar graph shows the distribution of critical and major alarms.", 850]
 
-Alarm Histogram:: Bar chart that shows the distribution of alarms by severity level.
+* *Alarms Table:* A table that displays alarms.
+The query for this panel must include an `asTableSummary()` transform, otherwise the panel will not display any data.
 +
-image::alarm-histogram.png[Horizontal bar graph showing distribution of critical and major alarms, 500]
+image::alarm-panel.png["An Alarms Table panel showing two tables: one displays unacknowledged alarms, and the other displays acknowledged alarms", 850]
 
-Alarm Table:: Displays alarms in table format.
+* *Filter panel:* A panel that lets you specify data to include in other panels.
```

Comment:
How does one configure a Filter Panel? What other kinds of panels will it apply to? How do you configure the other panels to honor the filter panel's selection?

Reply:
adding @synqotik here since he worked on the filter panel and might have a better answer than me. Although my understanding is Filter Panel only works with Alarm Table Panel where an alarm table check if there is a Filter panel in the dashboard and apply the filter to the Alarm Table panel....but I'm not too sure about that

Reply:
I'll take a look at this, with respect to the Filter Panel.

Reply:
Filter Panel overview above should read something like "A panel that allows you to filter entity data in other panels"

Reply:
I could help write up some documentation for the Filter Panel. Not sure if it's better to do it here, or else just open a separate ticket for it rather than hold this PR up.

Filter Panel applies to all other panels in the same dashboard which use the same Entity datasource (they actually have to be using the React Entity Datasource). Nothing else special has to be done in any other panels.

Anywhere in another panel where you have a query that gets entity data, e.g. all nodes or alarms, it will also be filtered by any active Filter Panel selections.

For example in the case of the Alarm Table Panel, if you have a Filter Panel in the same dashboard, you can add an alarm filter in the Filter Panel, including a specific alarm attribute like 'severity', and any items selected in the Filter Panel will limit ('filter') which alarms are displayed in in the Alarm Table Panel.

Reply:
@emarsh1995 sounds good! Yes we can open another ticket if needed.